concordia/settings_template.py

```diff
@@ -99,6 +99,7 @@
 ]
 
 RATELIMIT_VIEW = "concordia.views.ratelimit_view"
+RATELIMIT_BLOCK = False
```

Comment:
I'm wondering whether we should move the `RATELIMIT_BLOCK` down here so we can test it in staging, etc.

Reply:
`settings_prod` is used in dev, test, stage and prod environments.

Reply:
Jenkins builds the docker images which use `settings_prod` - Travis uses `settings_test` and local development runserver uses `settings_dev`

Reply:
I guess where I was going with that is that as our logic gets more complicated we really need to nail it down with tests but the code-coverage tracking should help with that, too.

Reply:
So our unit test execution runs with rate limiting totally disabled - https://github.com/LibraryOfCongress/concordia/blob/master/concordia/settings_test.py#L25

It seems to me that running unit tests with rate limiting enabled introduces a bunch of problems. We would need to change the configuration of the rate for test purposes.

Reply:
I think we should use the settings override decorator / context manager to selectively enable it only for the cases where we're actually testing this logic so we don't have random collateral damage.